renovate.json

```diff
@@ -0,0 +1,8 @@
+{
+  "automerge": true,
```

Comment:
Doesn't this pose a significant security risk given how this compiled code is run?

Reply:
Automerge wouldn't actually occur until we approve it, since there is currently branch protection on